autoload/neomake.vim

```diff
@@ -476,6 +485,14 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
             endif
         endif
         let status = a:data
+        if has_key(jobinfo, 'exit_callback')
+            try
+                let callback_dict = { 'status': status,
+                                    \ 'name': maker.name,
+                                    \ 'has_next': has_key(maker, 'next') }
+                exe 'call '.(jobinfo.exit_callback).'(callback_dict)'
```

Comment:
It would be nice to support functions too.

```
if type(jobinfo.exit_callback) == type('')
    let ExitCallback = function(jobinfo.exit_callback)
else
    let ExitCallback = jobinfo.exit_callback
endif
call ExitCallback(callback_dict)
```